2_Functions_Part_2.ipynb

```diff
@@ -77,9 +77,20 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                final_price = final_price + (prices.get(ingredient) * ingredients.get(ingredient) / 100)
</pre><p>always use the syntax</p><p>final_price += x </p><p>over </p><p>final_price = final_price + x</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week05-sagin132/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='135'/>

Reply:
<p>Ok</p>